src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -36,12 +41,18 @@ def environment_variables_format(value):
 register_cli_argument('container create', 'image', help='The container image name')
 register_cli_argument('container create', 'cpu', type=int, help='The required number of CPU cores of the containers')
 register_cli_argument('container create', 'memory', type=float, help='The required memory of the containers in GB')
-register_cli_argument('container create', 'os_type', help='The OS type of the containers', **enum_choice_list(OS_TYPES))
+register_cli_argument('container create', 'os_type', help='The OS type of the containers', **enum_choice_list(OperatingSystemTypes))
 register_cli_argument('container create', 'ip_address', help='The IP address type of the container group', **enum_choice_list(IP_ADDRESS_TYPES))
+register_cli_argument('container create', 'ports', type=int, nargs='+', help='The ports to open')
```

Comment:
Still don't mention the 5 port max in help.

Reply:
Like I mentioned, I'd rather not validate this in CLI. We might have special quota for certain subscriptions. We have validation on server side for this.

Reply:
Okay with all the comments here, I missed that one.

Reply:
@derekbekoe if you don't have any other comments, please help merge. Thanks!